docs/concepts/work-pools.md

```diff
@@ -0,0 +1,273 @@
+│ test-pool  │  process │ a51adf8c-58bb-4949-abe6-1b87af46eabd │ None              │
+└────────────┴──────────────────────────────────────┴───────────────────┘
+                       (**) denotes a paused pool
+                    ```
+</div>
+
+`prefect work-pool inspect` provides all configuration metadata for a specific work pool by ID.
+
+<div class="terminal">
+```bash
+$ prefect work-pool inspect 'test-pool'
+Workpool(
+    id='a51adf8c-58bb-4949-abe6-1b87af46eabd',
+    created='2 minutes ago',
+    updated='2 minutes ago',
+    name='test-pool',
+    filter=None,
+)
+```
+</div>
+
+`prefect work-pool preview` displays scheduled flow runs for a specific work pool by ID for the upcoming hour. The optional `--hours` flag lets you specify the number of hours to look ahead. 
+
+<div class="terminal">
+```bash
+$ prefect work-pool preview 'test-pool' --hours 12 
+┏━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃ Scheduled Star… ┃ Run ID                     ┃ Name         ┃ Deployment ID               ┃
+┡━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+│ 2022-02-26 06:… │ 741483d4-dc90-4913-b88d-0… │ messy-petrel │ 156edead-fe6a-4783-a618-21… │
+│ 2022-02-26 05:… │ 14e23a19-a51b-4833-9322-5… │ unselfish-g… │ 156edead-fe6a-4783-a618-21… │
+│ 2022-02-26 04:… │ deb44d4d-5fa2-4f70-a370-e… │ solid-ostri… │ 156edead-fe6a-4783-a618-21… │
+│ 2022-02-26 03:… │ 07374b5c-121f-4c8d-9105-b… │ sophisticat… │ 156edead-fe6a-4783-a618-21… │
+│ 2022-02-26 02:… │ 545bc975-b694-4ece-9def-8… │ gorgeous-mo… │ 156edead-fe6a-4783-a618-21… │
+│ 2022-02-26 01:… │ 704f2d67-9dfa-4fb8-9784-4… │ sassy-hedge… │ 156edead-fe6a-4783-a618-21… │
+│ 2022-02-26 00:… │ 691312f0-d142-4218-b617-a… │ sincere-moo… │ 156edead-fe6a-4783-a618-21… │
+│ 2022-02-25 23:… │ 7cb3ff96-606b-4d8c-8a33-4… │ curious-cat… │ 156edead-fe6a-4783-a618-21… │
+│ 2022-02-25 22:… │ 3ea559fe-cb34-43b0-8090-1… │ primitive-f… │ 156edead-fe6a-4783-a618-21… │
+│ 2022-02-25 21:… │ 96212e80-426d-4bf4-9c49-e… │ phenomenal-… │ 156edead-fe6a-4783-a618-21… │
+└─────────────────┴────────────────────────────┴──────────────┴─────────────────────────────┘
+                                   (**) denotes a late run
+```
+</div>
+
+### Pausing and deleting work pools
+
+A work pool can be paused at any time to stop the delivery of work to agents. Agents will not receive any work when polling a paused pool.
+
+To pause a work pool through the Prefect CLI, use the `prefect work-pool pause` command:
+
+<div class="terminal">
+```bash
+$ prefect work-pool pause 'test-pool'
+Paused work pool 'test-pool'
+```
+</div>
+
+To resume a work pool through the Prefect CLI, use the `prefect work-pool resume` command with the work pool name.
+
+To delete a work pool through the Prefect CLI, use the `prefect work-pool delete` command with the work pool name.
+
+### Work pool concurrency
```

Comment:
we need a section on work pool queue priority - that's the headline new feature here IMO